### PR TITLE
feat: add prop to set arrow positions on first time point out

### PR DESCRIPTION
**Pull request by scurker**
Added `arrowPosition` prop to `<FirstTimePointOut />` component to allow arrow positions [via the deque pattern library](https://pattern-library.dequelabs.com/components/first-time-point).

Ref: #54

### CODE REVIEW

File: demo/patterns/components/FirstTimePointOut/index.js [hunk 1]
```diff
@@ -24,13 +33,19 @@ import { FirstTimePointOut } from 'react-cauldron';
 
 const Demo = () => (
   <section>
-    <FirstTimePointOut headerId={'ftpo-head'}>
-      <h4 id={'ftpo-head'}>First time point out!</h4>
+    <h3>With Default Arrow</h3>
```
Comment (scurker) [inline demo/patterns/components/FirstTimePointOut/index.js]: @schne324 I think having demo examples can be improved long term without adding a bunch of cruft. Let's chat at some point.

Comment (schne324) [reply demo/patterns/components/FirstTimePointOut/index.js]: Agreed. I plan on doing #53 soon which will help

File: package.json [hunk 2]
```diff
@@ -59,7 +59,7 @@
     "classnames": "^2.2.5",
     "closest": "0.0.1",
     "css-loader": "^0.28.7",
-    "deque-pattern-library": "^2.5.1",
+    "deque-pattern-library": "^6.1.0",
```
[the remark this answers is not in the document]
Comment (schne324) [reply package.json]: Sorry about that...dqpl is versioned using semver via [standard-version](https://github.com/dequelabs/pattern-library/blob/master/package.json#L9)

If you look at https://github.com/dequelabs/pattern-library/blob/master/CHANGELOG.md you can find several *BREAKING CHANGES* notes but it _does_ look like https://github.com/dequelabs/pattern-library/pull/110 was missed in the generated changelog so I will manually update it

**Comment by schne324** [comment]
I'm not sure why ci is crying about jest not being found...[this _was_ working](https://circleci.com/gh/dequelabs/cauldron-react/310)

Any ideas?

**Comment by scurker** [comment]
@schne324 I don't normally use yarn, so the lock file had a hash version mismatch due to the upgraded dependency.

https://github.com/dequelabs/cauldron-react/blob/7f87eb578b93fa840906e950caed59b7257ee296/.circleci/config.yml#L13-L14

**Pull request merged**